website/settings/defaults.py

```diff
@@ -330,3 +330,5 @@ def parent_dir(path):
 assert (DRAFT_REGISTRATION_APPROVAL_PERIOD > EMBARGO_END_DATE_MIN), 'The draft registration approval period should be more than the minimum embargo end date.'
 
 PREREG_ADMIN_TAG = "prereg_admin"
+
+INSTITUTION_HIDE_UI_FLAG = True
```

Comment:
As suggested by Michael, this is a specifically and explicitly for ui hiding. The api endpoints don't need to be hidden (I would think)